bedrock/firefox/templates/firefox/channel/ios.html

```diff
@@ -6,10 +6,30 @@
 
 {% add_lang_files "firefox/channel/index" %}
 
-{% block page_desc %}{{_('Test beta versions of Firefox for iOS via Apple’s TestFlight program and help make our mobile browser for iPhone, iPad and iPod touch even better.')}}{% endblock %}
+{% block page_title %}
+{% if l10n_has_tag('firefox_title_meta_bug1345338') %}
+  {{ _('Try New Features in a Pre-Release iOS Browser | Firefox') }}
+{% else %}
+  {{_('Test beta versions of Firefox for iOS via Apple’s TestFlight program and help make our mobile browser for iPhone, iPad and iPod touch even better.')}}
+{% endif %}
+{% endblock %}
+
+{% block page_desc %}
+{% if l10n_has_tag('firefox_title_meta_bug1345338') %}
+  {{_('Experience cutting-edge features in a pre-release browser for iOS: Firefox Developer Edition, Firefox Beta and Firefox Nightly. Download now!')}}
```

Comment:
With apologies, I think we need to modify this copy -- mobile browsers have different channel names.

> Experience cutting-edge features in a pre-release browser for iOS via Apple’s TestFlight program. Install now!